ucm2/Intel/sof-essx8336/HiFi.conf

```diff
@@ -1,4 +1,35 @@
 SectionVerb {
+	EnableSequence [
+		# Disable all inputs / outputs
+		cset "name='Speaker Switch' off"
+		cset "name='Headphone Switch' off"
+		cset "name='Headset Mic Switch' off"
+		cset "name='Internal Mic Switch' off"
+		cset "name='DAC Mono Mix Switch' off"
+
+		# Setup muxes / switches
+		cset "name='Left Headphone Mixer Left DAC Switch' on"
+		cset "name='Right Headphone Mixer Right DAC Switch' on"
+		# Set digital mix mux to "dmic disable"
+		# That doesn't affect dmic, but other values mute headset mic
+		cset "name='Digital Mic Mux' 0"
+
+		# Set HP vol to 0 dB
+		cset "name='Headphone Playback Volume' 100%"
+		cset "name='Headphone Mixer Volume' 100%"
+
+		# Set DAC vol
+		cset "name='DAC Playback Volume' 60%"
+		# LDATA TO LDAC, RDATA TO RDAC
+		cset "name='DAC Source Mux' 0"
+
+		# Disable Auto Level Control
+		cset "name='ALC Capture Switch' off"
+
+		# Set capture vol
+		cset "name='ADC Capture Volume' 60%"
```

Comment:
the settings between line17 and 30 should probably be part of the boot sequence so that they are set even when UCM is not used to select a device?

Reply:
Ok, I'll move them to the boot sequence.